### ISSUE TITLE
WorldEdit causing errors in log and PlotSquared not letting players place blocks

### ISSUE THREAD

**Issue by almateoskott** [IntellectualSites/PlotSquared#2192]
**Debug paste link:** 

https://hastebin.com/fufezojife

**Description of the problem:** 
It worked fine in Minecraft 1.12.2.  After the 1.13.2 update, players are unable to place or destroy blocks in the Creative plot world.  I have tried several different 1.13.2 builds of PlotSquared.  When an opped account places a block, the block places.  In both cases, there is a flood of errors in the logs and it mentions WorldEdit.  That happens even if I just use WorldEdit itself in Survival--to replicate that I just mark and bound and then //set 1 . (But setting to anything causes the same issue).  If I remove PlotSquared and just use WorldEdit on its own, no problems.  I have tried with WorldEdit 7 for Bukkit 1.13+ from https://dev.bukkit.org/projects/worldedit/files . I tried Beta3 and Beta4.  Same thing, WE works fine until PlotSquared is installed.  I got the PlotSquared from here.  https://ci.athion.net/job/PlotSquared-Breaking/ .   The issue happens even if I rename my PlotSquared and WorldEdit settings directories and let them regenerate the config.

**How to replicate:** 

Install PlotSquared and WorldEdit.
Reboot server
//wand
Mark a few blocks
//set 1

Then the logs will flood.  Here is a sample.

----------
24.12 16:03:57 [Server] INFO ... 36 more 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.eventbus.EventHandler.handleEvent(EventHandler.java:73) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.eventbus.MethodEventHandler.dispatch(MethodEventHandler.java:58) ~[?:?] 24.12 16:03:57 [Server] INFO at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_111] 24.12 16:03:57 [Server] INFO at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_111] 24.12 16:03:57 [Server] INFO at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_111] 24.12 16:03:57 [Server] INFO at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_111] 24.12 16:03:57 [Server] INFO at com.plotsquared.listener.WESubscriber.onEditSession(WESubscriber.java:54) ~[?:?] 24.12 16:03:57 [Server] INFO Caused by: java.lang.NoClassDefFoundError: Could not initialize class com.plotsquared.listener.WEManager 24.12 16:03:57 [Server] INFO ... 36 more 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.eventbus.EventHandler.handleEvent(EventHandler.java:73) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.eventbus.MethodEventHandler.dispatch(MethodEventHandler.java:58) ~[?:?] 24.12 16:03:57 [Server] INFO at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_111] 24.12 16:03:57 [Server] INFO at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_111] 24.12 16:03:57 [Server] INFO at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_111] 24.12 16:03:57 [Server] INFO at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_111] 24.12 16:03:57 [Server] INFO Caused by: java.lang.reflect.InvocationTargetException 24.12 16:03:57 [Server] INFO at java.lang.Thread.run(Unknown Source) [?:1.8.0_111] 24.12 16:03:57 [Server] INFO at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:733) [Spigot.jar:git-Spigot-a1ba5fe-3359cb1] 24.12 16:03:57 [Server] INFO at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:835) [Spigot.jar:git-Spigot-a1ba5fe-3359cb1] 24.12 16:03:57 [Server] INFO at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:417) [Spigot.jar:git-Spigot-a1ba5fe-3359cb1] 24.12 16:03:57 [Server] INFO at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:900) [Spigot.jar:git-Spigot-a1ba5fe-3359cb1] 24.12 16:03:57 [Server] INFO at net.minecraft.server.v1_13_R2.SystemUtils.a(SourceFile:199) [Spigot.jar:git-Spigot-a1ba5fe-3359cb1] 24.12 16:03:57 [Server] INFO at java.util.concurrent.FutureTask.run(Unknown Source) [?:1.8.0_111] 24.12 16:03:57 [Server] INFO at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_111] 24.12 16:03:57 [Server] INFO at net.minecraft.server.v1_13_R2.PlayerConnectionUtils.a(SourceFile:10) ~[Spigot.jar:git-Spigot-a1ba5fe-3359cb1] 24.12 16:03:57 [Server] INFO at net.minecraft.server.v1_13_R2.PacketPlayInChat.a(PacketPlayInChat.java:1) ~[Spigot.jar:git-Spigot-a1ba5fe-3359cb1] 24.12 16:03:57 [Server] INFO at net.minecraft.server.v1_13_R2.PacketPlayInChat.a(PacketPlayInChat.java:45) ~[Spigot.jar:git-Spigot-a1ba5fe-3359cb1] 24.12 16:03:57 [Server] INFO at net.minecraft.server.v1_13_R2.PlayerConnection.a(PlayerConnection.java:1481) ~[Spigot.jar:git-Spigot-a1ba5fe-3359cb1] 24.12 16:03:57 [Server] INFO at net.minecraft.server.v1_13_R2.PlayerConnection.handleCommand(PlayerConnection.java:1646) ~[Spigot.jar:git-Spigot-a1ba5fe-3359cb1] 24.12 16:03:57 [Server] INFO at org.bukkit.craftbukkit.v1_13_R2.CraftServer.dispatchCommand(CraftServer.java:699) ~[Spigot.jar:git-Spigot-a1ba5fe-3359cb1] 24.12 16:03:57 [Server] INFO at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:141) ~[Spigot.jar:git-Spigot-a1ba5fe-3359cb1] 24.12 16:03:57 [Server] INFO at com.sk89q.bukkit.util.DynamicPluginCommand.execute(DynamicPluginCommand.java:54) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.bukkit.WorldEditPlugin.onCommand(WorldEditPlugin.java:231) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.eventbus.EventBus.post(EventBus.java:173) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.eventbus.EventBus.dispatch(EventBus.java:187) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.eventbus.EventHandler.handleEvent(EventHandler.java:73) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.eventbus.MethodEventHandler.dispatch(MethodEventHandler.java:58) ~[?:?] 24.12 16:03:57 [Server] INFO at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_111] 24.12 16:03:57 [Server] INFO at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_111] 24.12 16:03:57 [Server] INFO at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_111] 24.12 16:03:57 [Server] INFO at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_111] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.extension.platform.CommandManager.handleCommand(CommandManager.java:275) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.command.SimpleDispatcher.call(SimpleDispatcher.java:125) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.command.composition.LegacyCommandAdapter.call(LegacyCommandAdapter.java:52) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.command.composition.SelectionCommand.call(SelectionCommand.java:46) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.command.composition.SelectionCommand.call(SelectionCommand.java:74) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.LocalSession.createEditSession(LocalSession.java:858) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.EditSessionFactory$EditSessionFactoryImpl.getEditSession(EditSessionFactory.java:173) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.EditSession.<init>(EditSession.java:248) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.EditSession.wrapExtent(EditSession.java:271) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.eventbus.EventBus.post(EventBus.java:173) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.eventbus.EventBus.dispatch(EventBus.java:187) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.eventbus.EventHandler.handleEvent(EventHandler.java:75) ~[?:?] 24.12 16:03:57 [Server] INFO java.lang.reflect.InvocationTargetException: null 24.12 16:03:57 [Server] ERROR Could not dispatch event: com.sk89q.worldedit.event.extent.EditSessionEvent@3e840c1e to handler EventHandler{priority=VERY_EARLY} 24.12 16:03:57 [Server] INFO ... 36 more 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.eventbus.EventHandler.handleEvent(EventHandler.java:73) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.eventbus.MethodEventHandler.dispatch(MethodEventHandler.java:58) ~[?:?] 24.12 16:03:57 [Server] INFO at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_111] 24.12 16:03:57 [Server] INFO at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_111] 24.12 16:03:57 [Server] INFO at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_111] 24.12 16:03:57 [Server] INFO at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_111] 24.12 16:03:57 [Server] INFO at com.plotsquared.listener.WESubscriber.onEditSession(WESubscriber.java:54) ~[?:?] 24.12 16:03:57 [Server] INFO Caused by: java.lang.NoClassDefFoundError: Could not initialize class com.plotsquared.listener.WEManager 24.12 16:03:57 [Server] INFO ... 36 more 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.eventbus.EventHandler.handleEvent(EventHandler.java:73) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.eventbus.MethodEventHandler.dispatch(MethodEventHandler.java:58) ~[?:?] 24.12 16:03:57 [Server] INFO at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_111] 24.12 16:03:57 [Server] INFO at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_111] 24.12 16:03:57 [Server] INFO at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_111] 24.12 16:03:57 [Server] INFO at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_111] 24.12 16:03:57 [Server] INFO Caused by: java.lang.reflect.InvocationTargetException 24.12 16:03:57 [Server] INFO at java.lang.Thread.run(Unknown Source) [?:1.8.0_111] 24.12 16:03:57 [Server] INFO at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:733) [Spigot.jar:git-Spigot-a1ba5fe-3359cb1] 24.12 16:03:57 [Server] INFO at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:835) [Spigot.jar:git-Spigot-a1ba5fe-3359cb1] 24.12 16:03:57 [Server] INFO at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:417) [Spigot.jar:git-Spigot-a1ba5fe-3359cb1] 24.12 16:03:57 [Server] INFO at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:900) [Spigot.jar:git-Spigot-a1ba5fe-3359cb1] 24.12 16:03:57 [Server] INFO at net.minecraft.server.v1_13_R2.SystemUtils.a(SourceFile:199) [Spigot.jar:git-Spigot-a1ba5fe-3359cb1] 24.12 16:03:57 [Server] INFO at java.util.concurrent.FutureTask.run(Unknown Source) [?:1.8.0_111] 24.12 16:03:57 [Server] INFO at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_111] 24.12 16:03:57 [Server] INFO at net.minecraft.server.v1_13_R2.PlayerConnectionUtils.a(SourceFile:10) ~[Spigot.jar:git-Spigot-a1ba5fe-3359cb1] 24.12 16:03:57 [Server] INFO at net.minecraft.server.v1_13_R2.PacketPlayInChat.a(PacketPlayInChat.java:1) ~[Spigot.jar:git-Spigot-a1ba5fe-3359cb1] 24.12 16:03:57 [Server] INFO at net.minecraft.server.v1_13_R2.PacketPlayInChat.a(PacketPlayInChat.java:45) ~[Spigot.jar:git-Spigot-a1ba5fe-3359cb1] 24.12 16:03:57 [Server] INFO at net.minecraft.server.v1_13_R2.PlayerConnection.a(PlayerConnection.java:1481) ~[Spigot.jar:git-Spigot-a1ba5fe-3359cb1] 24.12 16:03:57 [Server] INFO at net.minecraft.server.v1_13_R2.PlayerConnection.handleCommand(PlayerConnection.java:1646) ~[Spigot.jar:git-Spigot-a1ba5fe-3359cb1] 24.12 16:03:57 [Server] INFO at org.bukkit.craftbukkit.v1_13_R2.CraftServer.dispatchCommand(CraftServer.java:699) ~[Spigot.jar:git-Spigot-a1ba5fe-3359cb1] 24.12 16:03:57 [Server] INFO at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:141) ~[Spigot.jar:git-Spigot-a1ba5fe-3359cb1] 24.12 16:03:57 [Server] INFO at com.sk89q.bukkit.util.DynamicPluginCommand.execute(DynamicPluginCommand.java:54) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.bukkit.WorldEditPlugin.onCommand(WorldEditPlugin.java:231) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.eventbus.EventBus.post(EventBus.java:173) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.eventbus.EventBus.dispatch(EventBus.java:187) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.eventbus.EventHandler.handleEvent(EventHandler.java:73) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.eventbus.MethodEventHandler.dispatch(MethodEventHandler.java:58) ~[?:?] 24.12 16:03:57 [Server] INFO at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_111] 24.12 16:03:57 [Server] INFO at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_111] 24.12 16:03:57 [Server] INFO at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_111] 24.12 16:03:57 [Server] INFO at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_111] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.extension.platform.CommandManager.handleCommand(CommandManager.java:275) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.command.SimpleDispatcher.call(SimpleDispatcher.java:125) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.command.composition.LegacyCommandAdapter.call(LegacyCommandAdapter.java:52) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.command.composition.SelectionCommand.call(SelectionCommand.java:46) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.command.composition.SelectionCommand.call(SelectionCommand.java:74) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.LocalSession.createEditSession(LocalSession.java:858) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.EditSessionFactory$EditSessionFactoryImpl.getEditSession(EditSessionFactory.java:173) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.EditSession.<init>(EditSession.java:242) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.EditSession.wrapExtent(EditSession.java:271) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.eventbus.EventBus.post(EventBus.java:173) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.eventbus.EventBus.dispatch(EventBus.java:187) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.eventbus.EventHandler.handleEvent(EventHandler.java:75) ~[?:?] 24.12 16:03:57 [Server] INFO java.lang.reflect.InvocationTargetException: null 24.12 16:03:57 [Server] ERROR Could not dispatch event: com.sk89q.worldedit.event.extent.EditSessionEvent@33d8a1f9 to handler EventHandler{priority=VERY_EARLY} 24.12 16:03:57 [Server] INFO ... 36 more 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.eventbus.EventHandler.handleEvent(EventHandler.java:73) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.eventbus.MethodEventHandler.dispatch(MethodEventHandler.java:58) ~[?:?] 24.12 16:03:57 [Server] INFO at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_111] 24.12 16:03:57 [Server] INFO at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_111] 24.12 16:03:57 [Server] INFO at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_111] 24.12 16:03:57 [Server] INFO at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_111] 24.12 16:03:57 [Server] INFO at com.plotsquared.listener.WESubscriber.onEditSession(WESubscriber.java:54) ~[?:?] 24.12 16:03:57 [Server] INFO Caused by: java.lang.NoClassDefFoundError: Could not initialize class com.plotsquared.listener.WEManager 24.12 16:03:57 [Server] INFO ... 36 more 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.eventbus.EventHandler.handleEvent(EventHandler.java:73) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.eventbus.MethodEventHandler.dispatch(MethodEventHandler.java:58) ~[?:?] 24.12 16:03:57 [Server] INFO at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_111] 24.12 16:03:57 [Server] INFO at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_111] 24.12 16:03:57 [Server] INFO at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_111] 24.12 16:03:57 [Server] INFO at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_111] 24.12 16:03:57 [Server] INFO Caused by: java.lang.reflect.InvocationTargetException 24.12 16:03:57 [Server] INFO at java.lang.Thread.run(Unknown Source) [?:1.8.0_111] 24.12 16:03:57 [Server] INFO at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:733) [Spigot.jar:git-Spigot-a1ba5fe-3359cb1] 24.12 16:03:57 [Server] INFO at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:835) [Spigot.jar:git-Spigot-a1ba5fe-3359cb1] 24.12 16:03:57 [Server] INFO at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:417) [Spigot.jar:git-Spigot-a1ba5fe-3359cb1] 24.12 16:03:57 [Server] INFO at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:900) [Spigot.jar:git-Spigot-a1ba5fe-3359cb1] 24.12 16:03:57 [Server] INFO at net.minecraft.server.v1_13_R2.SystemUtils.a(SourceFile:199) [Spigot.jar:git-Spigot-a1ba5fe-3359cb1] 24.12 16:03:57 [Server] INFO at java.util.concurrent.FutureTask.run(Unknown Source) [?:1.8.0_111] 24.12 16:03:57 [Server] INFO at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_111] 24.12 16:03:57 [Server] INFO at net.minecraft.server.v1_13_R2.PlayerConnectionUtils.a(SourceFile:10) ~[Spigot.jar:git-Spigot-a1ba5fe-3359cb1] 24.12 16:03:57 [Server] INFO at net.minecraft.server.v1_13_R2.PacketPlayInChat.a(PacketPlayInChat.java:1) ~[Spigot.jar:git-Spigot-a1ba5fe-3359cb1] 24.12 16:03:57 [Server] INFO at net.minecraft.server.v1_13_R2.PacketPlayInChat.a(PacketPlayInChat.java:45) ~[Spigot.jar:git-Spigot-a1ba5fe-3359cb1] 24.12 16:03:57 [Server] INFO at net.minecraft.server.v1_13_R2.PlayerConnection.a(PlayerConnection.java:1481) ~[Spigot.jar:git-Spigot-a1ba5fe-3359cb1] 24.12 16:03:57 [Server] INFO at net.minecraft.server.v1_13_R2.PlayerConnection.handleCommand(PlayerConnection.java:1646) ~[Spigot.jar:git-Spigot-a1ba5fe-3359cb1] 24.12 16:03:57 [Server] INFO at org.bukkit.craftbukkit.v1_13_R2.CraftServer.dispatchCommand(CraftServer.java:699) ~[Spigot.jar:git-Spigot-a1ba5fe-3359cb1] 24.12 16:03:57 [Server] INFO at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:141) ~[Spigot.jar:git-Spigot-a1ba5fe-3359cb1] 24.12 16:03:57 [Server] INFO at com.sk89q.bukkit.util.DynamicPluginCommand.execute(DynamicPluginCommand.java:54) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.bukkit.WorldEditPlugin.onCommand(WorldEditPlugin.java:231) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.eventbus.EventBus.post(EventBus.java:173) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.eventbus.EventBus.dispatch(EventBus.java:187) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.eventbus.EventHandler.handleEvent(EventHandler.java:73) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.eventbus.MethodEventHandler.dispatch(MethodEventHandler.java:58) ~[?:?] 24.12 16:03:57 [Server] INFO at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_111] 24.12 16:03:57 [Server] INFO at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_111] 24.12 16:03:57 [Server] INFO at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_111] 24.12 16:03:57 [Server] INFO at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_111] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.extension.platform.CommandManager.handleCommand(CommandManager.java:275) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.command.SimpleDispatcher.call(SimpleDispatcher.java:125) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.command.composition.LegacyCommandAdapter.call(LegacyCommandAdapter.java:52) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.command.composition.SelectionCommand.call(SelectionCommand.java:46) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.command.composition.SelectionCommand.call(SelectionCommand.java:74) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.LocalSession.createEditSession(LocalSession.java:858) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.EditSessionFactory$EditSessionFactoryImpl.getEditSession(EditSessionFactory.java:173) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.EditSession.<init>(EditSession.java:235) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.EditSession.wrapExtent(EditSession.java:271) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.eventbus.EventBus.post(EventBus.java:173) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.eventbus.EventBus.dispatch(EventBus.java:187) ~[?:?] 24.12 16:03:57 [Server] INFO at com.sk89q.worldedit.util.eventbus.EventHandler.handleEvent(EventHandler.java:75) ~[?:?] 24.12 16:03:57 [Server] INFO java.lang.reflect.InvocationTargetException: null 24.12 16:03:57 [Server] ERROR Could not dispatch event: com.sk89q.worldedit.event.extent.EditSessionEvent@40459b80 to handler EventHandler{priority=VERY_EARLY}

--------------

**Checklist**:

- [X] I included a `/plot debugpaste` link
- [X] I made sure there are no duplicates of this report [(Use Search)](https://github.com/IntellectualSites/PlotSquared/issues?utf8=%E2%9C%93&q=is%3Aissue)
- [X] I made sure I am using an up-to-date version of PlotSquared
- [X] I Made sure the bug/error is not caused by any other plugin

**Comment by Sauilitired** [comment]
`I have tried several different 1.13.2 builds of PlotSquared`
The only 1.13.2 builds are the ones in the "breaking" build instance, and those have not been released yet. You can try them out [here](https://ci.athion.net/job/PlotSquared-Breaking/). If you are updating from an earlier version use [this](https://github.com/IntellectualSites/PlotSquared/wiki/Installation#updating-to-113) guide. However, we do not yet guarantee that those builds will work 100% as intended. I highly recommend that you join our Discord if you haven't already, as we're able to communicate better over there.

**Comment by Sauilitired** [comment]
This issue has been resolved in the new pre-release.

**Comment by almateoskott** [comment]
Thanks.  It's working fine on my server now.  Just want to share something.  The flood in the logs was resolved with the update to PlotSquared-Bukkit-4.18.12.25-edc7583-2105.jar . The issue with the blocks not placing was not caused by PlotSquared.  I traced that to of all things PreciousStones.  It was preventing all placement, not just use of sponges in my plots/Creative world.  Left at its defaults of:
  prevent-place-everywhere:
  - creative_world
  prevent-destroy-everywhere:
  - creative_world
It actually lets players place in any world but one named exactly "creative_world".  So the block placing part reported above was not caused by a PlotSquared bug.  The mess-up in the logs part was.
